tests/acceptance/features/apiGraph/activities.feature

```diff
@@ -445,3 +445,231 @@ Feature: check activities
+                      "pattern": "^%user_id_pattern%$"
+                    },
+                    "template": {
+                      "type": "object",
+                      "required": ["message", "variables"],
+                      "properties": {
+                        "message": {
+                          "const": "{user} renamed {oldResource} to {resource}"
+                        },
+                        "variables": {
+                          "type": "object",
+                          "required": ["oldResource", "resource", "user"],
+                          "properties": {
+                            "oldResource": {
+                              "type": "object",
+                              "required": ["id", "name"],
+                              "properties": {
+                                "id": {
+                                  "const": ""
+                                },
+                                "name": {
+                                  "const": "textfile0.txt"
+                                }
+                              }
+                            },
+                            "resource": {
+                              "type": "object",
+                              "required": ["id", "name"],
+                              "properties": {
+                                "id": {
+                                  "type": "string",
+                                  "pattern": "^%file_id_pattern%$"
+                                },
+                                "name": {
+                                  "const": "renamed.txt"
+                                }
+                              }
+                            },
+                            "user": {
+                              "type": "object",
+                              "required": ["id", "displayName"],
+                              "properties": {
+                                "id": {
+                                  "type": "string",
+                                  "pattern": "^%user_id_pattern%$"
+                                },
+                                "displayName": {
+                                  "const": "Alice"
+                                }
+                              }
+                            }
+                          }
+                        }
+                      }
+                    },
+                    "times": {
+                      "type": "object",
+                      "required": ["recordedTime"],
+                      "properties": {
+                        "recordedTime": {
+                          "type": "string",
+                          "format": "date-time"
+                        }
+                      }
+                    }
+                  }
+                }
+              ]
+            }
+          }
+        }
+      }
+      """
+
+  @issue-9712
+  Scenario: check rename activity for a folder
```

Comment:
Can we do file and folder in a single scenario?
- rename file
- rename folder
- check file activity
- check folder activity

Reply:
Similar for other scenarios as well